tests/unit/test_ingestor_server/test_ingestor_library.py

```
@@ -113,10 +113,27 @@ def ingestor(
     @pytest.fixture
     def mock_nvingest_upload_doc(self, ingestor):
         """Mock the __nvingest_upload_doc method."""
+        async def mock_nvingest(*args, **kwargs):
```

Ruff (0.14.5)

Missing return type annotation for private function `mock_nvingest`

(ANN202)